src/batch/src/executor/delete.rs

```diff
@@ -91,9 +92,7 @@ impl DeleteExecutor {
         let rows_deleted = try_join_all(notifiers)
             .await
             .map_err(|_| {
-                RwError::from(ErrorCode::InternalError(
-                    "failed to wait chunks to be written".to_owned(),
-                ))
+                BatchExecutorError::Internal(anyhow!("failed to wait chunks to be written"))
```

Comment:
Change name to BatchError?

Reply:
for now, I put error.rs in src/batch/executor/  so I named it BatchExecutorError.
Is good to change the name to BatchError? (there are other crate(batch/execution,batch/rpc) in batch.

Reply:
+1 for BatchError, I don't see we need error for each component. And why not move it to src dir?